core/clients/key_flow_continuous_refresh.go

```diff
@@ -0,0 +1,126 @@
+package clients
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+
+	"github.com/stackitcloud/stackit-sdk-go/core/oapierror"
+)
+
+var (
+	defaultTimeStartBeforeTokenExpiration = 30 * time.Minute
+	defaultTimeBetweenContextCheck        = time.Second
+	defaultTimeBetweenTries               = 5 * time.Minute
+)
+
+// Continuously refreshes the token of a key flow, retrying if the token API returns 5xx errrors. Writes to stderr when it terminates.
+//
+// To terminate this routine, close the context in keyFlow.config.TokenRefreshInBackgroundContext.
```

Comment:
the comment still has the old field name

Reply:
Fixed